### ISSUE TITLE
Data shown in US units even though HUBOT_FORECAST_UNITS is set to "SI" or even unset

### ISSUE THREAD

**Issue by tkothe** [jeffbyrnes/hubot-forecast#16]
I'm running this on a hubot on heroku. Right now, temperature is shown in US units (Fahrenheit), even though I have set to SI units (or even unset HUBOT_FORECAST_UNITS).

Here's my config: 

```
HUBOT_FORECAST_KEY:         XXXXX
HUBOT_FORECAST_ROOM:        my-room
HUBOT_FORECAST_UNITS:       SI
HUBOT_LATITUDE:             53.551
HUBOT_LONGITUDE:            9.99
```

I have also tried with unsetting HUBOT_FORECAST_UNITS.

This is the output I am getting: 

```
Here is your weather report…
Currently: Partly Cloudy 47.52°C
Today: Light rain and breezy starting this evening, continuing until tomorrow morning.
Coming week: Light rain today through Saturday, with temperatures peaking at 51°F on Wednesday.
```

It's clearly not "47.52°C" in Hamburg, Germany :) Also, it's interesting that the "coming week" section shows "51°F"
In the heroku console I see

```
2015-11-29T12:47:05.562443+00:00 app[web.1]: [Forecast] Requesting forecast data: https://api.forecast.io/forecast/<REMOVED>/53.551,9.99?units=us&exclude=flags
```

which obviously should have "units=si" instead :disappointed:

**Comment by jeffbyrnes** [comment]
Thanks for spotting this. I haven’t done nearly enough testing here, but this is a nasty bug for everyone who’s not an American. I’ll see what I can do!

**Comment by jeffbyrnes** [comment]
Just cannot find the cycles to properly tackle this, sorry :(

**Comment by jeffbyrnes** [comment]
And of course, managed to carve out a bit of time to fix this. Turned out to be a really straightforward fix. Tests well on my end, but @tkothe, please verify v0.5.1 is all good.

**Comment by TheReal1604** [comment]
@jeffbyrnes Can you please upload your version v0.5.1 (or the fix for this) to npm? I installed your very nice script via npm and got the problem above (set to SI but get US units).

Thx in advance!

**Comment by jeffbyrnes** [comment]
@TheReal1604 oh derp, sorry about that! I’ve published v0.5.1 just now.

**Comment by TheReal1604** [comment]
@jeffbyrnes Thank you!
